NETCORE/test/IntegrationTests.Tests/RequestCollectionTest.cs

```diff
@@ -9,16 +9,21 @@
 using System.Threading.Tasks;
 using Xunit;
 using Xunit.Abstractions;
+#if NETCOREAPP2_1
+using IntegrationTests.WebApp._2._1;
+#else
+using IntegrationTests.WebApp._3._1;
+#endif
 
 namespace IntegrationTests.Tests
 {
-    public class BasicTestWithCorrelation : IClassFixture<CustomWebApplicationFactory<IntegrationTests.WebApp.Startup>>
+    public class RequestCollectionTest : IClassFixture<CustomWebApplicationFactory<Startup>>
```

Comment:
This is where I plan to migrate all other tests into! This is just refactoring in this PR. actual test migration is next!